[CTOR-656~] refresh Azure Database for MySQL doc page (#3408)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/cloud-azure-database-mysql.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Overview

Azure Database for MySQL allows you to easily set up scalable and secure high
availability MySQL instances.

## Pack assets

### Templates

The Monitoring Connector **Azure Database for MySQL** brings a host template:

* **Cloud-Azure-Database-MySQL-custom**

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="Cloud-Azure-Database-MySQL-custom" label="Cloud-Azure-Database-MySQL-custom">

| Service Alias  | Service Template                                     | Service Description                                   |
|:---------------|:-----------------------------------------------------|:------------------------------------------------------|

> | Connections    | Cloud-Azure-Database-MySQL-Connections-Api-custom    | Check Azure Database for MySQL connections statistics |

Reviewer: ```suggestion
| Connections    | Cloud-Azure-Database-MySQL-Connections-Api-custom    | Check Azure Database for MySQL connection statistics |
```

| Cpu            | Cloud-Azure-Database-MySQL-Cpu-Api-custom            | Check Azure Database for MySQL CPU usage              |

> | IO-Consumption | Cloud-Azure-Database-MySQL-IO-Consumption-Api-custom | Check Azure Database for MySQL IO usage               |

Reviewer: ```suggestion
| IO-Consumption | Cloud-Azure-Database-MySQL-IO-Consumption-Api-custom | Check Azure Database for MySQL I/O usage               |
```

| Memory         | Cloud-Azure-Database-MySQL-Memory-Api-custom         | Check Azure Database for MySQL memory usage           |
| Replication    | Cloud-Azure-Database-MySQL-Replication-Api-custom    | Check Azure Database for MySQL replication status     |
| Storage        | Cloud-Azure-Database-MySQL-Storage-Api-custom        | Check Azure Database for MySQL storage statistics     |
| Traffic        | Cloud-Azure-Database-MySQL-Traffic-Api-custom        | Check Azure Database for MySQL network usage          |

> The services listed above are created automatically when the **Cloud-Azure-Database-MySQL-custom** host template is used.

</TabItem>
<TabItem value="Not attached to a host template" label="Not attached to a host template">

| Service Alias | Service Template                              | Service Description                          |
|:--------------|:----------------------------------------------|:---------------------------------------------|

> | Queries       | Cloud-Azure-Database-MySQL-Queries-Api-custom | Check Azure Database for MySQL queries count |

Reviewer: ```suggestion
| Queries       | Cloud-Azure-Database-MySQL-Queries-Api-custom | Check the number of queries to Azure Database for MySQL |
```

| AZURERESOURCEGROUP | Set resource group (required if resource's name is used)                                                                   |                   | X           |
| AZURERESOURCETYPE  | Set resource type (default: 'servers'). Can be 'servers', 'flexibleServers'                                                |                   | X           |
| AZURESUBSCRIPTION  | Set Azure subscription (required if logged to several subscriptions)                                                       |                   | X           |
| AZURETENANT        | Set Azure tenant ID                                                                                                        |                   | X           |
| PROXYURL           | Proxy URL if any                                                                                                           |                   |             |
| EXTRAOPTIONS       | Any extra option you may want to add to every command (a --verbose flag for example). All options are listed [here](#available-options).                       |                   |             |

> > Set the following options in the EXTRAOPTIONS macro if you are monitoring resource from Microsoft Azure operated by 21Vianet (Azure China):

Reviewer: ```suggestion
> Set the following options in the EXTRAOPTIONS macro if you are monitoring resources from Microsoft Azure operated by 21Vianet (Azure China):
```

## How to check in the CLI that the configuration is OK and what are the main options for?

Once the plugin is installed, log into your Centreon poller's CLI using the
**centreon-engine** user account (`su - centreon-engine`). Test that the connector
is able to monitor an Azure Instance using a command like this one (replace the sample values by yours):

```bash
/usr/lib/centreon/plugins/centreon_azure_database_mysql_api.pl \
	--plugin=cloud::azure::database::mysql::plugin \
	--mode=traffic \
	--custommode='api' \
	--resource='' \
	--resource-group='' \
	--resource-type='' \
	--subscription='xxx' \
	--tenant='xxx' \
	--client-id='xxx' \
	--client-secret='xxx' \
	--proxyurl=''  \
	--filter-metric='' \
	--filter-dimension='' \
	--timeframe='900' \
	--interval='PT5M' \
	--aggregation='Average' \
	--warning-traffic-out='' \
	--critical-traffic-out='' \
	--warning-traffic-in='' \
	--critical-traffic-in=''
```

The expected command output is shown below:

```bash

> OK: Network Out: 66 B Network In: 83 B | 'azmysql.traffic.out.bytes'=66B;;;0;'azmysql.traffic.in.bytes'=83B;;;0;

Reviewer 2: ```suggestion
OK: Network Out: 66 B Network In: 83 B | 'azmysql.traffic.out.bytes'=66B;;;0; 'azmysql.traffic.in.bytes'=83B;;;0;
```
